<launch>
  <node 
    pkg="dr_lidar_calib"
    type="multi_dr_lidar_calib"
    name="multi_dr_lidar_calib"
    args="$(find dr_lidar_calib)/config/config_dr_lidar_calib.yaml
    $(find dr_lidar_calib)/result/multi_dr_lidar/
    1"
    output="screen"
  />
  <!-- <node 
    pkg="rviz"
    type="rviz"
    name="rviz"
    args="-d $(find dr_lidar_calib)/rviz_cfg/calib_indoor.rviz"
  /> -->
</launch> 